### PR TITLE
manage commands part 1

### PR DESCRIPTION
**Pull request by MrSerge01**
## added
- /manage bam (fake ban)

## major changes
- made /manage ban work
- made /manage kick work

## minor changes
- /info user (changed `.setTitle()` to `.setAuthor()`)
- /manage clean (changed up the embed a bit)

## improvements needed
- better embeds on all manage commands
- making the "needed permissions" error work

# credits
- [ThatFrogDev](https://github.com/ThatFrogDev)

### CODE REVIEW

File: src/commands/manage/ban.js [hunk 1]
```diff
@@ -18,31 +18,49 @@ module.exports = class Ban {
...
       );
   }
-  
-  run(interaction) {
-    if (interaction.user.id !== OWNER && !ADMIN.includes(interaction.user.id)) return;
 
-    const client = interaction.client;
+  async run(interaction) {
+    let errorEmbed = new EmbedBuilder().setColor(getColor(0));
+    // if (interaction.user.id !== OWNER && !ADMIN.includes(interaction.user.id)) {
```
Comment (ThatFrogDev) [inline src/commands/manage/ban.js:24]: Better uncomment this so people cant abuse it

File: src/commands/manage/ban.js [hunk 2]
```diff
@@ -18,31 +18,49 @@ module.exports = class Ban {
...
     const user = interaction.options.getUser("user");
     const reason = interaction.options.getString("reason");
-    const channel = client.channels.cache.get("979337971159420928");
+    const channel = interaction.client.channels.cache.get("979337971159420928");
+    const member = interaction.member;
+    const everyone = member.guild.roles.everyone;
+    const allMembers = await member.guild.members.fetch();
+    const allRoles = await member.guild.roles.fetch();
+    const selectedMember = allMembers.filter(m => m.user.id === user.id).get(user.id);
+    const yes = allRoles.filter(r => r !== everyone && selectedMember._roles.includes(r.id)).sort((a, b) => Math.max([a[1].rawPosition, b[1].rawPosition]));
```
Comment (ThatFrogDev) [inline src/commands/manage/ban.js:37]: ```const yes``` lmfao

**Review by ThatFrogDev — APPROVED** [review]
Very nice changes, LGTM. I left some comments but apart from that I think we can merge.

File: src/commands/manage/ban.js (hunk 2 above)
Comment (MrSerge01) [reply src/commands/manage/ban.js:37]: temp name

File: src/commands/manage/ban.js (hunk 1 above)
Comment (MrSerge01) [reply src/commands/manage/ban.js:24]: tmrw

**Pull request merged**